UnityProject/Assets/Scripts/Systems/Antagonists/AlienPlayer.cs

```diff
@@ -1603,8 +1600,7 @@ private void OnSpawnFromGhostRole(PlayerInfo player)
 			playerTookOver = player;
 
 			//Transfer player chosen into body
-			PlayerSpawn.ServerTransferPlayerToNewBody(player.Connection, player.Mind, gameObject,
-				Event.PlayerSpawned, null);
+			//PlayerSpawn.TransferAccountToSpawnedMind(player, ); //TODO Need reference to mind!!!!!!!!!!!
```

Comment:
at this point can you just do a:

```cs
//TODO: (Bod/Mind) Need to reference the player's mind.
```

This way when we search for todos inside of Rider of visual studio we can know what we're looking at

Reply:
think I fixed this , need to test I do